iati/core/resources/test_data/202/valid_startswith.xml

```diff
@@ -1,6 +1,21 @@
 <?xml version="1.0"?>
-
+<!--all elements and attributes contain the specified prefixing string-->
 <root_element>
-  <reporting-org ref="text"></reporting-org><!-- iati-identifier starts with expected prefix -->
-  <iati-identifier>text-1234</iati-identifier>
+  <condition></condition><!--condition case-->
+  <element1>prefix1234</element1><!--single path-->
```

Comment:
Suggest mixing up the suffix through the various tests to show that it's not just matching the string `prefix1234`.

Reply:
good idea

Reply:
done